public/app/features/plugins/sql/components/visual-query-builder/AwesomeQueryBuilder.tsx

```diff
@@ -202,36 +204,74 @@ export const raqbConfig: Config = {
 
 export type { Config };
 
+const noop = () => '';
+
+const isSqlFormatOp = (func: unknown): func is SqlFormatOperator => {
+  return typeof func === 'function';
+};
+
 function getCustomOperators(config: BasicConfig) {
   const { ...supportedOperators } = config.operators;
-  const noop = () => '';
+
   // IN operator expects array, override IN formatter for multi-value variables
-  const sqlFormatInOp = supportedOperators[Op.IN].sqlFormatOp || noop;
+  const sqlFormatInOpOrNoop = () => {
+    const sqlFormatOp = supportedOperators[Op.IN].sqlFormatOp;
+    if (isSqlFormatOp(sqlFormatOp)) {
+      return sqlFormatOp;
+    }
+    return noop;
+  };
```

Comment:
Narrowing down the type to `SqlFormatOperator` https://github.com/ukrbublik/react-awesome-query-builder/blob/master/packages/core/modules/index.d.ts#L810